### PR TITLE
Re-detailed the Korath Raider

### PR DESCRIPTION
**Pull request by Red-57**
----------------------
**Content (Artwork)**

## Summary
Re-detailed the Korath Raider sprite and added reverse thrusting to it.
(old-new)
raider
raider
raider
raider

due to a general popularity on discord the weird jumbled boxes were removed.

## Save File
NA

## PR Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I uploaded the necessary image, blend, and texture assets here: 
(these are the ones with the jumbled boxes)
[raider assets.zip](https://github.com/endless-sky/endless-sky/files/5578141/raider.assets.zip)
(these are the ones without the jumbled boxes)
[raider assets.zip](https://github.com/endless-sky/endless-sky/files/5647181/raider.assets.zip)

 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/156

### CODE REVIEW

**Comment by Zitchas** [comment]
Interesting. Side by side before and after pictures would probably be helpful, too.

**Comment by ravenshining** [comment]
Reverse thrusters, yes, because a reverse Korath thruster is planned, steering flares, no.

I could be persuaded to say yes to adding steering flares to ships, but not if they aren't balanced (unless we got some cool mechanics to change how ships turn.)  For the steering flares you added here to work it needs a second set firing the opposite direction on the opposite quadrants of the shp, for example on the forward section of the nacelles, like this
raiderflare

File: copyright [hunk 1]
```diff
@@ -866,6 +866,13 @@ Copyright: Tommaso Becca
 License: CC-BY-SA-4.0
 Comment: Derived from works by ESA/Hubble & NASA (under the same license)
 
+Files:
+ images/ship/raider
+ images/thumbnail/raider
```
Comment (Anarchist2) [inline copyright:871]:
```suggestion
 images/ship/raider*
 images/thumbnail/raider*
```

File: data/korath/korath ships.txt [hunk 2]
```diff
@@ -35,20 +35,38 @@ ship "Korath Raider"
 		"Korath Grab-Strike" 4
 		"Korath Banisher"
 		"Korath Warder"
-		
+
```
Comment (Anarchist2) [inline data/korath/korath ships.txt:38]: Don't change the whitespace.

**Comment by Anarchist2** [comment]
Also, don't have dimensions for images that are odd, or else they'll show up blurry ingame. Ideally, stick to dimensions that are multiples of ten.

File: data/korath/korath ships.txt (hunk 2 above)
Comment (Red-57) [reply data/korath/korath ships.txt:38]: I didn't mean too computer did it automatically, fixing now

File: copyright (hunk 1 above)
Comment (Red-57) [reply copyright:871]: ok

File: data/korath/korath ships.txt (hunk 2 above)
Comment (Red-57) [reply data/korath/korath ships.txt]: I think that I fixed the errors.

**Comment by Red-57** [comment]
I re-did the steering engines so they are more balenced
Screen Shot 2020-11-22 at 8 10 35 AM

**Comment by ravenshining** [comment]
That looks better.  Are you going to add nozzles for the rears, too?

I'm no expert on vectors, but it looks like the rears need more vertical component.  Angle 140 and -140 would be a perfect inverse of the fronts.  Also, you could experiment with fitting them inboard of the nacelles at that angle.

**Comment by Amazinite** [comment]
I'm not much a fan of the steering flares on this ship. Note that just because steering flares are a thing doesn't mean that all ships will now end up having steering flares. And as Anarchist mentioned, you have issues with the image dimensions here. You've actually shrunk the Raider a tiny bit. Otherwise, I dig it. A subtle change overall to the sprite, but it looks good.

**Review by Amazinite — CHANGES_REQUESTED** [review]
Fix the image size to match the preexisting image, and I'd suggest ditching the steering flares.

**Comment by Red-57** [comment]
should I trash the steering engine things or just the sprite that shows up?

**Comment by Amazinite** [comment]
Up to you whether you change the model. Speaking of which though, I also notice from the thumbnail images that the Raider has appeared to have gone on a bit of a diet; it's noticeably thinner with this new model compared to the old. Personally, I prefer the slightly chunkier current thumbnail.

**Comment by Red-57** [comment]
I did not change the overall thickness of any of the models, it was like that when I found it. I can try to fix that though.

**Comment by ravenshining** [comment]
Can you put the crystal bits back in?  That might restore the "chunkiness."  I realise that would hide the forward nozzles, but those can be moved, especially since they aren't being used now.

**Comment by Red-57** [comment]
most people on discord said that they didn't like them, so I got rid of them. If a people want me to I can add them back though.

**Comment by ravenshining** [comment]
So I took this and ran with it a bit further, and after testing a number of designs came up with something that seems to have satisfied the people who don't like the crystals.  Model is basically done, need to tweak the lighting a bit still, but when I have, shall I PR your branch to update this PR, or would you rather close this and have me open a new one based on my forking it?

Screenshot_20201208_000551

**Comment by Pointedstick** [comment]
FWIW I think the steering flares are kind of cool. The fact that other ships don't have them makes this one seem especially unique and cool.

**Comment by ravenshining** [comment]
I agree, but not on doing them piecemeal, or having just one random ship here and there have them  I could add steering thrusters to all Korath ships as I go through them, one by one, and then have a separate PR that adds the flares to all of them at once.

**Comment by ravenshining** [comment]
Okay, I am done, either you can merge the PR I sent you @Red-57 like you asked on Discord, or if you don't want to wait for him to do that @Amazinite , you can check out my branch here: [https://github.com/ravenshining/endless-sky/tree/raiderrefresh](https://github.com/ravenshining/endless-sky/tree/raiderrefresh)

**Comment by ravenshining** [comment]
Oh, right, updated assets here:

[raider assets.zip](https://github.com/endless-sky/endless-sky/files/5686838/raider.assets.zip)

Also note I adjusted some hardpoints, most notably moving the fighters so they emerge from the back of the pods backwards, rather than weirdly forwards from the middle of the pods.

**Pull request merged**